packages/context/test/unit/ui5-model.test.ts

```diff
@@ -11,20 +32,72 @@ import {
   getCacheFilePath,
   getCacheFolder,
   negotiateVersionWithFetcher,
+  VersionMapJsonType,
 } from "../../src/ui5-model";
 import { FetchResponse } from "@ui5-language-assistant/language-server";
+import { fetch } from "@ui5-language-assistant/logic-utils";
+import * as logicUtils from "@ui5-language-assistant/logic-utils";
+import { getVersionJsonUrl } from "../../src/utils";
+import semverMinSatisfying from "semver/ranges/min-satisfying";
+import { Response } from "node-fetch";
+
+const GET_MODEL_TIMEOUT = 30000;
```

Comment:
any specific reason to have this timeout? By default it pick this https://github.com/SAP/ui5-language-assistant/blob/master/jest.setup.js#L1